content/blog/2022/02/camel315-whatsnew/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Apache Camel 3.15 What's New"
+date: 2022-02-04
+authors: [davsclaus]
+categories: ["Releases"]
+preview: Details of what we have done in the Camel 3.15 release.
+---
+
+Apache Camel 3.15 has just been [released](/blog/2022/02/RELEASE-3.15.0/).
```

Comment:
Should this point to the release (`/releases/release-3.15.0/`)?

Reply:
It was a copy of the 3.14 where we linked to Gregors release announcements when he finalize the release
This blog will come after his blog posts.